tests/test.api.js

```diff
@@ -61,12 +61,26 @@ lab.test('getMultiplePages', async() => {
   server.route({
     path: '/api/pages/{slug}',
     method: 'get',
-    handler: (request, h) => ({ content: request.params.slug })
+    handler: (request, h) => ({
+      content: {
+        common: { test: 123 },
```

Comment:
this is a bit confusing because common, page1 and slug aren't typically inside a page